projects/go-lib/src/lib/components/go-datepicker/calendar-views.scss

```diff
@@ -4,50 +4,56 @@
   display: inline-flex;
   height: 1rem;
   margin-top: 0.5rem;
+  padding: 0 .5rem;
   width: 100%;
   padding-bottom: 1.5rem;
+  justify-content: space-between;
 }
 
-.go-calendar__header--left {
+.go-calendar__header--center {
   display: flex;
-  flex: 1;
-  margin-left: .5rem;
+}
 
-  &:hover, &:active, &:focus {
-    background: $theme-light-bg-hover;
-    cursor: pointer;
-  }
+.go-calendar__header--center-offset {
+  margin-left: .5rem;
 }
 
-.go-calendar__header--right {
+.go-calendar__header--outside {
   align-items: center;
   display: flex;
-  padding-top: .5rem;
+  padding: .5rem;
 }
 
 .go-calendar__date {
-  line-height: 2rem;
-  text-align: right;
-  margin: 5px;
+  line-height: 1.5rem;
+  padding: 0 .25rem;
+}
+
+.go-calendar__date--tall {
+  padding: .5rem .25rem;
 }
 
 .go-calendar__header-dropdown {
   margin-top: -.5rem;
 }
 
 .go-calendar__body--header {
-  border-bottom: 1px solid black;
   padding-bottom: .5rem;
   padding-top: .5rem;
+  font-size: 0.875rem;
+}
+
+.go-calendar__body--header-border {
+  border-bottom: 1px solid $theme-light-border;
 }
 
 .go-calendar__cell {
   border-style: hidden;
   border-radius: 1.5rem;
-  font-size: .75rem;
-  height: 1.75rem;
-  width: 90%;
-  padding: 5%;
+  font-size: 0.75rem;
+  padding: 0px;
+  height: 1.375rem;
+  width: 2.563rem;
```

Comment:
This measurement seems real gross. Does it need to be exactly this width? Could we make it something a little more rounded?

Reply:
I rounded it to 2.5rem 🤷‍♂